directed_work/04_vertex/README.md

```diff
@@ -0,0 +1,162 @@
+# Directed work 4 [Sprint 4, W08]: Vertex
```

Comment:
To handover:
- Codes
- Screenshot of pipeline
- Screenshot of output data
- Screenshot of performance metrics

Reply:
Other idea: Ask to launch 2-3 pipelines with different parameters (or data) and show the performance metrics (in artifacts) and deployed models
https://docs.google.com/presentation/d/1hbSsnvZVdvDPxqzv7cnj_m4CciSVUN0xWIWtKRwZbwM/edit#slide=id.g3226daf73fc_0_1549